contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -5070,6 +5071,39 @@ static char *get_oid_type_string(int type_oid){
 	return type_string;
 }
 
+static int64 get_identity_into_args(Node *node){
+
+	int64 val;
+	Const *con;
+	FuncExpr *fxpr;
+	ListCell * lc;
+	
+	switch (nodeTag(node))
+	{
+		case T_Const:
+			con = (Const *)node;
+			val = con->constvalue;
+			break;
+		case T_FuncExpr:
+			fxpr = (FuncExpr *)node;
+			foreach(lc, fxpr->args)
+			{
+				Node *n = (Node *)lfirst(lc);
+				if (IsA(n, Const))
+				{
+					con = (Const *)n;
+					val = (int64) DatumGetInt64(con->constvalue);
+					break;
+				}
```

Comment:
add an else in here and throw out some error

Reply:
okay